docs/Developers/Newcomers_Guide.rst

```diff
@@ -208,27 +208,27 @@ Now you can start working on it!
     You can do amazing stuff using corobo.
 
         * Issue assigning as stated earlier.
-        * File issues::
```

Comment:
Different issue removing the colons :)

Reply:
@kriti21 has an issue been created about this yet?
If not, please create one.  if you describe clearly what needs to be done, and show them this PR, it is a newcomer issue, and we'll find a newcomer to do it, and you need to remove this stuff from your PR.

Reply:
But adding the extra colon makes the background of all the following corobo commands grey colored. Is is okay to push a PR like that (not looking good) ? @jayvdb

Reply:
Create a separate issue.  The extra colon is a syntax *bug*, and is unrelated to your issue.